aiohttp/connector.py

```diff
@@ -467,30 +481,57 @@ def family(self):
         return self._family
 
     @property
+    def cache_dns(self):
```

Comment:
`cache_dns` assumes some action, here is "to cache dns". But for property either `dns_cache` or `cached_dns` are better names.